src/EnergyPlus/EconomicLifeCycleCost.cc

```diff
@@ -1496,7 +1496,13 @@ namespace EconomicLifeCycleCost {
 
          for (int iCashFlow = 1; iCashFlow <= numCashFlow; ++iCashFlow) {
             if (CashFlow(iCashFlow).pvKind == pvkEnergy) {
-                curResource = CashFlow(iCashFlow).Resource - ResourceTypeInitialOffset;
+                // make sure this is not water
+                int curResource_iRT = CashFlow(iCashFlow).Resource;
+                if (CashFlow(iCashFlow).Resource == iRT_Water ||
+                    (CashFlow(iCashFlow).Resource >= iRT_OnSiteWater && CashFlow(iCashFlow).Resource <= iRT_Condensate)) {
+                    continue;
```

Comment:
Just a thought. It would be a little faster if these iRT variables were reordered, with water types at the end, so that only
```
if (CashFlow(iCashFlow).Resource >= iRT_Water) continue;
```

could be used. But probably wouldn't save that much since this code is only executed at end? of simulation.

Reply:
It would make sense to optimize this if it was in a timestep but since this is not it probably is not worth the effort.